### PR TITLE
Add warning about not using external_inputs in proto

### PR DESCRIPTION
**Pull request by klecki**
The use of this field is deprecated.

Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

#### Why we need this PR?
Add a comment about no longer used filed in proto

#### What happened in this PR? 
 - What solution was applied:
     Comment added
 - Affected modules and functionalities:
     DALI proto (but not really)
 - Key points relevant for the review:
     Comment
 - Validation and testing:
     CI
 - Documentation (including examples):
     Sort of

**JIRA TASK**: *[Use DALI-XXXX or NA]*

### CODE REVIEW

File: dali/pipeline/proto/dali.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2017-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (banasraf) [inline dali/pipeline/proto/dali.proto:1]: I think it's rather `2017, 2021`

**Review by szalpal — APPROVED** [review]
👍🏻

File: dali/pipeline/proto/dali.proto (hunk 1 above)
Comment (klecki) [reply dali/pipeline/proto/dali.proto:1]: If anything it should be 2017-2018, 2021, but my script doesn't handle that.

Comment (banasraf) [reply dali/pipeline/proto/dali.proto:1]: Okay, let's not complicate it. 👍

**Review by banasraf — APPROVED** [review]

**Comment by klecki** [comment]
!build

**Review by JanuszL — APPROVED** [review]

**Comment by klecki** [comment]
!build

**Comment by klecki** [comment]
!build

**Pull request merged**